Copilot/README.md

```diff
@@ -87,9 +87,7 @@ npm run start
 ### 1. Clone this repository and install the SDK into your React repo
 
 ```bash
-git clone -b feature/copilot https://github.com/microsoft/SharePoint-Embedded-Samples
-
-cd SharePoint-Embedded-Samples/Samples/spa-azurefunction
+# Install the SDK with npm
 
 npm install "https://download.microsoft.com/download/1315a30d-fe00-45b3-a149-d3235201f8ce/microsoft-sharepointembedded-copilotchat-react-1.0.2.tgz"
```

Comment:
Make the versions consistent everywhere. This one is `1.0.2` but in a change below you made something else `1.0.3`. Ensure it's the same.